### PR TITLE
Roben/grid fixes

### PR DESCRIPTION
**Pull request by ron-b**

### CODE REVIEW

File: src/utils/gridUtils.ts [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    config: MergedGridConfig;
+}
+
+const parseToTableData = (grid: Grid): TableData => {
+    const mergedGrid: MergedGrid = {
+        config: {
+            promoteHeaders: grid.config?.promoteHeaders ?? false,
+            adjustColumnNames: grid.config?.adjustColumnNames ?? true,
+        },
+        data: grid.data.map((row) => row.map((value) => value.toString())),
```
Comment (mabezen) [inline src/utils/gridUtils.ts:22]: What if the value is null, we had a fix that null become empty string

Comment (ron-b) [reply src/utils/gridUtils.ts:22]: Can you reproduce passing null with the type checks that exist?

Comment (mabezen) [reply src/utils/gridUtils.ts:22]: As far as I see parseToTableData run on a raw data, which can be null, it was easly reproduced using the sandbox by sending null value in the json.
Could you please double check \ add test

**Review by mabezen — APPROVED** [review]

File: src/utils/gridUtils.ts (hunk 1 above)
Comment (ron-b) [reply src/utils/gridUtils.ts]: See latest commit, fixed and changed tests environment to support this.

**Review by mabezen — APPROVED** [review]

**Pull request merged**